lib/call-stack-to-nix.nix

```diff
@@ -15,9 +15,16 @@
 , resolverSha256 ? null
 , materialized ? null # Location of a materialized copy of the nix files
 , checkMaterialization ? null # If true the nix files will be generated used to check plan-sha256 and material
-, nix-tools ? pkgs.haskell-nix.internal-nix-tools # When building stack projects we use the internal nix-tools (compiled with a fixed GHC version)
-, ... }:
+, nix-tools ? null
+, evalSystem ? null
+, ... }@args:
 let
+  pkgs = defaults.pkgs.haskell-nix.evalPackagesFor evalSystem;
```

Comment:
can we call this `evalPkgs`? I think it's really helpful to keep the distinction obvious, in fact currently this is shadowing the `pkgs` from line 8, which seems suspicious?